ddl/constant.go

```diff
@@ -38,8 +38,8 @@ const (
 	HistoryTableID = meta.MaxInt48 - 3
 	// MDLTableID is the table ID of `tidb_mdl_info`.
 	MDLTableID = meta.MaxInt48 - 4
-	// BackfillTableID is the table ID of `tidb_ddl_backfill`.
-	BackfillTableID = meta.MaxInt48 - 5
+	// BackgroundSubtasksTableID is the table ID of `tidb_background_subtasks`.
+	BackgroundSubtasksTableID = meta.MaxInt48 - 5
 	// BackfillHistoryTableID is the table ID of `tidb_ddl_backfill_history`.
 	BackfillHistoryTableID = meta.MaxInt48 - 6
```

Comment:
This 
```suggestion
	BackgroundSubTaskHistory = meta.MaxInt48 - 6
```

Reply:
Please also change this history table according

Reply:
Done